### PR TITLE
USHIFT-300: microshift: skip sig-imageregistry tests

### PR DESCRIPTION
**Pull request by pacevedom**
Skip sig-imageregistry tests if image.openshift.io API is not registered.

### CODE REVIEW

**Comment by pacevedom** [comment]
/hold
Need to rework.

**Comment by pacevedom** [comment]
/unhold
Ready for review
/assign @ingvagabund

File: test/extended/images/info.go [hunk 1]
```diff
@@ -21,6 +22,17 @@ var _ = g.Describe("[sig-imageregistry][Feature:ImageInfo] Image info", func() {
 		}
 	})
 
+	g.BeforeEach(func() {
+		microShift, err := exutil.IsMicroShiftDeployment(oc)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		if microShift {
+			// Test works as oc image info does not require any APIs, but the pod from
+			// where this is executed is a local image to the cluster, which is not
```
Comment (ingvagabund) [inline test/extended/images/info.go:30]: Any way to pull the image to the cluster?

Comment (pacevedom) [reply test/extended/images/info.go:30]: We would need to promote an image with oc inside to a public registry, not sure about that. Also, I wonder if this test should be in sig-cli, there are no checks in this one besides the pod coming up.

**Comment by pacevedom** [comment]
/hold
Need to get rid of microshift checks

**Comment by pacevedom** [comment]
/unhold
Using apigroups now.

File: test/extended/images/info.go [hunk 2]
```diff
@@ -23,7 +23,7 @@ var _ = g.Describe("[sig-imageregistry][Feature:ImageInfo] Image info", func() {
 
 	oc = exutil.NewCLIWithPodSecurityLevel("image-info", admissionapi.LevelBaseline)
 
-	g.It("should display information about images", func() {
+	g.It("should display information about images [apigroup:image.openshift.io]", func() {
```
Comment (ingvagabund) [inline test/extended/images/info.go:26]: Does the image apigroup requirement come from `cliPodWithPullSecret`? If so, is builder SA also implying build apigroup?

Comment (pacevedom) [reply test/extended/images/info.go:26]: The image this test uses holds `oc` binary in it and it comes from the internal registry. This is only working when this apigroup is installed.

**Review by soltysh — APPROVED** [review]
/approve

**Pull request merged**